client/src/components/Login/index.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import {
+  Grid,
+  Box,
+  Typography,
+  TextField,
+  Button,
+  FormControl,
+  FormHelperText,
+} from '@material-ui/core';
+import Style from './style';
+
+function Login(props) {
+  const classes = Style();
+
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [message, setMessage] = useState('');
+
+  const handleForm = (e) => {
+    e.preventDefault();
+    const data = { email, password };
+
```

Comment:
add validation here to check values, email format .... you can using this validation to check the email form

`import XRegExp from 'xregexp';`
`const validateEmail = (email) => {
  const re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
  return re.test(String(email).toLowerCase());
};
`